README.md

```diff
@@ -4,7 +4,23 @@ Simple reusable python resources for digital publishing.
 
 ## Installation
 
-**TODO**
+Follow these steps to install the package on your local machine:
```

Comment:
This is correct information but there's a much easier way to accomplish this. Have a look at this: https://www.geeksforgeeks.org/how-to-install-a-python-package-from-a-github-repository/

You should be able to get this down to something _like_ `pip install git+http://<something, cant remember>`.

Literally a one liner to install the python package from a github repo in the same way you would from pypi.

Take some time on understanding that one, its a phenomanally useful trick for a python dev to know.

From memory you should be able to specify a branch in there as well, so should be able to install from your dev branch to try it out - have a play 😄